answer/naka1998/JS/20/concat.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        const ary1=['a','b'];
+        const ary2=['c','d'];
+        const ary3 = ary1.concat(ary2);
+        let result = ary3.join("','");
+        console.log("['"+result+"']");
```

Comment:
MUST：演算子前後のスペース

nits: 変数と文字列の結合は、テンプレートリテラル(バッククォート)というものを使うことで、簡潔に書けます。

```suggestion
        console.log(`[${result}]`);
```